src/components/Button/Button.tsx

```diff
@@ -2,10 +2,12 @@ import { Icon } from "@/components";
 import { IconName } from "@/components/Icon/types";
 import styled from "styled-components";
 
-type ButtonType = "primary" | "secondary";
-type Alignment = "left";
+type ButtonType = "primary" | "secondary" | "danger";
+type Alignment = "default" | "left";
```

Comment:
@gjones we should use the true value instead of `default`, in this case `center` and then assign `center` as the default value in the props.

This is because we might change the default value in the future (e.g. make it left) and because one might not know what value `default` stands for, which might cause confusion